### PR TITLE
Restructured the code as per OCM CLI repo

### PR DESCRIPTION
**Pull request by tirthct**
**Changed**
- Added verbs before the resources in command
- Modified readme file

**Tested**
- Verbs commands should work properly

### CODE REVIEW

File: cmd/ocm-support/find/cmd.go [hunk 1]
```diff
@@ -0,0 +1,22 @@
+package find
```
Comment (cristianoveiga) [inline cmd/ocm-support/find/cmd.go:1]: are we keeping the command as **find**, not **get**?

File: cmd/ocm-support/find/registryCredentials/cmd.go [hunk 2]
```diff
@@ -1,4 +1,4 @@
-package show
+package registry_credentials
```
Comment (cristianoveiga) [inline cmd/ocm-support/find/registryCredentials/cmd.go:1]: following the ocm-cli, I they don't seem to use dashes or camel case for composed words. Should we do the same?

https://github.com/openshift-online/ocm-cli/blob/main/cmd/ocm/list/machinepool/cmd.go

We would have to change the folder names too.

File: README.md [hunk 3]
```diff
@@ -122,65 +113,68 @@ The following flags are available for `registryCredentials delete`:
...
+* Find an account and include its roles in the results `ocm support find accounts [accountID] --fetchRoles`
+* Find an account and include its labels in the results `ocm support find accounts [accountID] --fetchLabels`
+* Find an account and include its capabilities in the results `ocm support find accounts [accountID] --fetchCapabilities`
+* Find all accounts from an organization `ocm support find accounts [organizationID] --all`
+* Find an organization by its externalID: `ocm support find organizations [organizationExternalID]`
+* Find an organization and include its subscriptions: `ocm support find organizations [organizationID] --fetchSubscriptions`
+* Find an organization and include its labels `ocm support find organizations [organizationID] --fetchLabels --fetchCapabilities`
+* Find an organization and include its capabilities `ocm support find organizations [organizationID] --fetchCapabilities`
+* Find an organization and include its quota: `ocm support find organizations [organizationID] --fetchQuota`
+* Show registry credentials for a specific account `ocm support find registryCredentials [accountID]`
```
Comment (cristianoveiga) [inline README.md:180]: Shouldn't it be get registry credentials?

**Review by cristianoveiga — CHANGES_REQUESTED** [review]

File: cmd/ocm-support/create/registryCredentials/cmd.go [hunk 4]
```diff
@@ -9,12 +9,13 @@ import (
 	"github.com/openshift-online/ocm-support-cli/pkg/access_token"
 )
 
-// Cmd represents the version command
-var Cmd = &cobra.Command{
-	Use:   "create",
-	Short: "Creates registry credentials for the current account.",
-	Long:  "Created registry credentials for the current account.",
-	RunE:  run,
+// CmdCreateRegistryCredentials represents the version command
```
Comment (cristianoveiga) [inline cmd/ocm-support/create/registryCredentials/cmd.go:12]: This comment needs to be updated.

File: cmd/ocm-support/find/cmd.go (hunk 1 above)
Comment (tirthct) [reply cmd/ocm-support/find/cmd.go:1]: Missed it. Changed to **get**

**Comment by cristianoveiga** [comment]
LGTM

**Pull request merged**